core/server/lib/members/subscriptions/index.js

```diff
@@ -66,4 +66,22 @@ module.exports = class PaymentProcessorService {
             return this._processors[metadata.adapter].getSubscription(member, metadata);
         });
     }
+
+    removeSubscription(member, metadata) {
```

Comment:
`removeSubscription` is not needed for member deletion as `removeCustomer` automatically takes care of it, but extended files to expose already existing remove subscription method for future use.